website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,99 @@
+﻿
+---
+title: "Intro to Karpenter Concepts"
+linkTitle: "Concepts"
+weight: 20
+resources:
+- src: "**fig1*.png"
+- src: "**fig2*.png"
+---
+
+Kubernetes is designed to schedule new pods onto existing nodes. However, when
+existing nodes get full, pods are stuck pending.
+
+Karpenter provisions new capacity in response to pending pods that cannot
+be scheduled onto existing nodes. 
+Karpenter understands the instance types available from cloud providers. Karpenter’s provisioner intelligently selects new instances to start,
+using rich data from both podspecs and the cloud provider instance APIs. 
+
+Karpenter balances responding promptly to un-schedulable pods and making
+efficient provisioning decisions. Karpenter is more responsive than
+cluster autoscaler, while additionally considering podspec labels and cloud
+provider availability zones. Specifically, the commonly used Kubernetes
+properties such as labels, taints, affinity (node, pod), and anti-affinity are
+supported.

```

Comment:
I think you can drop the "(node, pod)" here

Also, 'are supported' feels like the wrong turn of phrase. 'are considered' seems redundant, but something like that might be more correct